using-mobile-extensions/adobe-journey-optimizer/android-messaging-push-improvements.md

```diff
@@ -0,0 +1,102 @@
+# Android Messaging Push Notification Improvements
+
+Messaging extension 1.1.0 introduces Messaging extension handled push notification creation and tracking. This new functionality provides a convenient way to display and track interactions with Adobe Journey Optimizer (AJO) created push notifications. The steps below will serve as a guide to enabling the new features provided in Messaging 1.1.0.
+
+{% hint style="info" %}
+
+The usage of these features is optional and no code changes are required if not using them.
+
+{% endhint %}
+
+When using Messaging handled push notification interaction tracking, the Messaging extension must also be used to create the push notification. To provide some additional flexibility in this scenario, two public interfaces are provided which allow a custom push notification factory (`IMessagingPushNotificationFactory`) or a custom notification image downloader (`IMessagingImageDownloader`) instance to be set. After these interfaces are implemented on the app side, they can be set via the new `setPushNotificationFactory` and `setPushImageDownloader` API.
+
+{% hint style="info" %}
+
+If the Messaging extension will be used to handle push notification interactions, ensure that your custom push notification factory sets a `PendingIntent` object for the content intent (`notificationBuilder.setContentIntent()`), delete intent (`notificationBuilder.setDeleteIntent()`), or button(s) when creating a push notification.
+
+{% endhint %}
+
+##### To get started with the Messaging 1.1.0 push notification features follow these steps:
+
+#### Step 1: Add the Messaging receivers in the App manifest
+
+Within your App's `AndroidManifest.xml` file, add a [manifest-declared receiver](https://developer.android.com/guide/components/broadcasts#manifest-declared-receivers) inside the application tag to subscribe to broadcast actions sent by the Messaging extension. A second manifest-declared receiver must be added to listen for notification interactions if Messaging handled push notification interaction tracking will be used:
```

Comment:
App can be lower case, since it's more generally as an "application", in context here.